core/types/data_container.cpp

```diff
@@ -202,6 +208,9 @@ String DataContainer::get_property_hint_name(const PropertyHint &p_hint) {
 		case PROPERTY_HINT_MAX: {
 			ERR_FAIL_V_MSG("", "Invalid property hint type.");
 		} break;
+		case PROPERTY_HINT_LOCALE_ID: {
+			return "Locale Id";
```

Comment:
Optionally we can do LocaleID to stylistically match with objectID

Reply:
I don't mind to change this style, as long as it's human-readable and consistent.